config/config/sylius_template_events.php

```diff
@@ -17,6 +17,10 @@
                         'template' => '@CommerceWeaversSyliusTpayPlugin/shop/order/pay/_card.html.twig',
                         'priority' => 10,
                     ],
+                    'pay-by-link' => [
```

Comment:
```suggestion
                    'pay_by_link' => [
```

to match the line `36`